### PR TITLE
Correct engines declaration

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by aredridel** [comment]
Whoops, this duplicates #137 more or less — I'm not sure which is correct, 12 or 0.12

**Comment by BlakeEddie** [comment]
https://github.com/medikoo/memoizee/pull/137 seems likely to be correct not really sure

**Review by quochung-bic — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -106,9 +106,9 @@
 		"prettify": "prettier --write --ignore-path .gitignore \"**/*.{css,html,js,json,md,yaml,yml}\"",
 		"prettify:updated": "pipe-git-updated ---base=main -ext=css --ext=html --ext=js --ext=json --ext=md --ext=yaml --ext=yml -- prettier --write",
 		"test": "tad"
-	},
```
Comment (medikoo) [inline package.json:109]: This change should not be here

**Review by medikoo — CHANGES_REQUESTED** [review]
@aredridel thanks for the effort, still this one actually breaks syntax

File: package.json (hunk 1 above)
Comment (aredridel) [reply package.json:109]: Ah, sorry about that! Fat fingered using github's web editor.

Comment (medikoo) [reply package.json:109]: No worries, change in other line was good. Anyway I've pushed it on my own and it's released now.

Again, thanks for helping on that

**Pull request closed without merging**